modules/imap/site.css

```diff
@@ -217,11 +217,14 @@
 .msg_parts td {
   padding: 3px;
   padding-right: 10px;
-  white-space: nowrap;
+  white-space: nowrap !important;
```

Comment:
Please avoid !important overrides - it makes theme development or integration css updates (like in Tiki) hard to bypass these overrides.